docs/config.rst

```diff
@@ -145,6 +145,18 @@ Those configuration options are documented below:
             includePaths: [/https?:\/\/getsentry\.com/, /https?:\/\/cdn\.getsentry\.com/]
         }
 
+.. describe:: collectWindowErrors
+
+  By default Raven collects all uncaught errors, that has been caught by `onerror` handler.
```

Comment:
I just realised that `uncaught`/`caught` wording... would appreciate native speaker input.

Reply:
"uncaught" sounds good to me, but if I were to tweak things, I would say:

> By default, Raven collects all uncaught errors that have been caught by the `onerror` handler. You can disable this behaviour by setting this flag to `false`. This may be useful when you only want to manually handle errors using the `captureMessage` and `captureException`.

Reply:
Thanks @crccheck!